src/main/java/seedu/duke/command/ViewTimetableCommand.java

```diff
@@ -17,9 +18,17 @@
 
 public class ViewTimetableCommand extends Command {
     public static final String COMMAND_WORD = "view";
+    private boolean showFancy;
+    private boolean showSimple;
 
-    public ViewTimetableCommand(String[] input) {
-        super(input);
+    public ViewTimetableCommand(String input) {
+        super(input.split("\\s+"));
+        var params = Parser.parseParams(input);
+        showFancy = params.containsKey("fancy");
+        showSimple = params.containsKey("simple");
+        // if (showFancy && showSimple) {
```

Comment:
Shouldn't this be kept since the user might enter both fancy and simple? If fancy has a higher priority when both keywords appear then you can remove these commented-out code.

Reply:
Will be uncommented when exception handling is ready.